Small Enterprise Hub updates (#1752)
huggingface/hub-docs: docs/hub/enterprise-hub-datasets.md

This feature is part of the <a href="https://huggingface.co/enterprise">Enterprise Hub</a>.
</Tip>

> Data Studio is enabled on private datasets owned by an Enterprise Hub organization.

Reviewer: ```suggestion
Data Studio is enabled on private datasets under your Enterprise Hub organization.
```

Data Studio allows teams to understand their data and to help them build better data processing and filtering for AI. This powerful viewer allows you to explore dataset content, inspect data distributions, filter by values, search for keywords, or even run SQL queries on your data without leaving your browser.

> [More information about Data Studio →](./datasets-viewer)

Reviewer: ```suggestion
More information about [Data Studio](./datasets-viewer).
```
